server.js

```diff
@@ -20,11 +13,38 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+	res.send('Golden Globe Winners');
+});
+
+//Fetches data from the API//
+app.get('/goldenGlobes', (req, res) => {
+	res.status(200).json(goldenGlobesData);
+});
+
+//Films by year//
+app.get('/goldenGlobes/year_award/:year_award', (req, res) => {
+	const filmByYear = goldenGlobesData.find(
+		(goldenGlobes) => goldenGlobes.year_award === req.params.year_award
+	);
+	if (!goldenGlobesData) {
+		res.status(404).json('Not found');
+	} else {
+		res.status(200).json({ data: filmByYear, success: true });
+	}
+});
+
+//Shows specific film and if it won or not//
+app.get('/goldenGlobes/win/:showName', (req, res) => {
+	const showName = req.params.showName;
+
+	let goldenGlobesWinners = goldenGlobesData.find(
+		(item) => item.nominee === showName
+	);
+	res.status(200).json({ data: goldenGlobesWinners.win, success: true });
```

Comment:
if I write the name of the show with the first letter being lowercase it becomes an error, you could use toLowerCase to fix that :)